CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Version 1.17.0
+* Release date: January 25, 2023
+* Release status: GA
+
+### What's new in 1.17.0
+* BREAKING CHANGE - Encryption is now Enabled by Default - https://github.com/microsoft/vscode-mssql/pull/17484
+	- Moving to Microsoft.Data.SqlClient 5.0.1 dependency with STS Update
+	- Connections should now be more sercure but some saved connections may be broken
+	- See https://aka.ms/vscodemssql-connection for more information. 
+* Introduced HostNameInCertificate Connection Property.
```

Comment:
@cheenamalhotra Do we have a link to docs to explain this further? Even just the SqlClient docs would be helpful

Reply:
the link is to a blog post that is going to go live tomorrow

Reply:
@erpett Charles is asking about line 12, not line 11

Reply:
Oh! I thought it was just because the link isnt live yet. Is there something you all would like to see me add to that line? Another possibility it to make it another sub-bullet with the group above since it was part of that same big PR

Reply:
* Introduced [HostNameInCertificate](https://learn.microsoft.com/dotnet/api/microsoft.data.sqlclient.sqlconnectionstringbuilder.hostnameincertificate#microsoft-data-sqlclient-sqlconnectionstringbuilder-hostnameincertificate) connection property.



(adds a link to the API docs for the property added)

Reply:
```suggestion
* Introduced 'HostNameInCertificate' Connection Property.
    - This property enables users to set the host name to use when validating the server certificate for the connection.
    - When not specified, the server name from the Data Source is used for certificate validation.
```
You can optionally link, https://learn.microsoft.com/dotnet/api/microsoft.data.sqlclient.sqlconnectionstringbuilder.hostnameincertificate, but it doesn't have much to offer, than what I added above.